test/providers/ipc_cuda_prov_consumer.c

```diff
@@ -22,23 +22,76 @@ int main(int argc, char *argv[]) {
     }
 
     int port = atoi(argv[1]);
+    CUdevice hDevice = -1;
+    CUcontext hContext = NULL;
 
-    cuda_memory_provider_params_t cu_params =
-        create_cuda_prov_params(UMF_MEMORY_TYPE_DEVICE);
+    int ret = get_cuda_device(&hDevice);
+    if (ret != 0) {
+        fprintf(stderr, "get_cuda_device() failed!\n");
+        return -1;
+    }
+
+    ret = create_context(hDevice, &hContext);
+    if (ret != 0) {
+        fprintf(stderr, "create_context() failed!\n");
+        return -1;
+    }
+
+    umf_cuda_memory_provider_params_handle_t cu_params = NULL;
+    umf_result_t umf_result = umfCudaMemoryProviderParamsCreate(&cu_params);
+    if (umf_result != UMF_RESULT_SUCCESS) {
+        fprintf(stderr, "Failed to create level zero params!\n");
```

Comment:
`[...] CUDA memory provider params!`, same with the rest of the error messages in the consumer and producer.

Reply:
Done